modules/ossm-enabling-sidecar-injection.adoc

```diff
@@ -0,0 +1,141 @@
+// Module included in the following assemblies:
+// install/ossm-sidecar-injection
+
+:_mod-docs-content-type: PROCEDURE
+[id="ossm-enabling-sidecar-injection_{context}"]
+= Enabling sidecar injection
+
+The sidecar injection process in {istio} uses label-based rules to determine if a pod should be injected with a sidecar. If a namespace has both the `istio-injection` label and the `istio.io/rev` label, the `istio-injection` label takes precedence, treating the namespace as part of the default revision.
+
+The sidecar injector follows the following logic:
+
+* If either the `istio-injection` label or the `sidecar.istio.io/inject` label is set to disabled, the pod is not injected.
+* If any of the following labels are enabled the pod is injected with a sidecar: `istio-injection`, `sidecar.istio.io/inject`, or `istio.io/rev`.
+
+.Prerequisites
+
+* The {SMProduct} operator is installed
+* An {istio} Container Network Interface (CNI) resource is created
+
+.Procedure
+
+. Create namespace called `istio-system`: 
++
+[source,terminal]
+----
+$ oc create ns istio-system
+----
+
+. Create a YAML file named `istio.yaml` with the name as `default`:
++
+[source,yaml,subs="attributes,verbatim"]
+----
+kind: Istio
+apiVersion: sailoperator.io/v1alpha1
+metadata:
+  name: default
+spec:
+  namespace: istio-system
+  updateStrategy:
+    type: InPlace
+  version: v1.23.0
+----
+
+. Apply the YAML file by running the following command:
++
+[source,terminal]
+----
+$ oc apply -f istio.yaml
+----
+
+. Wait for {istio} to get ready by running the following command:
++
+[source,terminal]
+----
+$ oc wait --for=condition=Ready istios/default -n istio-system
+----
+
+. Deploy the `sleep` application by running the following command:
++
+[source,terminal]
+----
+$ oc apply -f https://raw.githubusercontent.com/istio/istio/release-1.23/samples/sleep/sleep.yaml
+----
+
+. Verify that the deployment has a single container by running the folowing command:
++
+[source,terminal]
+----
+$ oc get deployment -o wide
+----
++
+You should see output similar to the following example:
++
+.Example output
+[source,terminal]
+----
+NAME    READY   UP-TO-DATE   AVAILABLE   AGE   CONTAINERS   IMAGES            SELECTOR
+sleep   1/1     1            1           16s   sleep        curlimages/curl   app=sleep
+----
```

Comment:
Even when a pod has sidecar, the output of `oc get deployment` will show only a single container.  Personally, I feel that this section (L65-L79) is too verbose/redundant and can be removed as we are capturing the notes on how to check the containers below using `oc get pod` command.

CC: @luksa @dgn for their thoughts.

Reply:
Okay I am removing this section (L65-L79).